source/Playnite.DesktopApp/ViewModels/SettingsViewModel.cs

```diff
@@ -233,6 +234,73 @@ public RelayCommand<object> WindowClosingCommand
             });
         }
 
+        public RelayCommand AddSortingNameRemovedArticle
+        {
+            get => new RelayCommand(() =>
+            {
+                var res = dialogs.SelectString(
+                    resources.GetString(LOC.EnterName),
+                    resources.GetString(LOC.AddNewItem),
+                    string.Empty);
+                if (res.Result && !res.SelectedString.IsNullOrEmpty())
+                {
+                    if (settings.GameSortingNameRemovedArticles.Any(a => a.Equals(res.SelectedString, StringComparison.InvariantCultureIgnoreCase)))
+                    {
+                        dialogs.ShowErrorMessage(resources.GetString(LOC.ItemAlreadyExists), string.Empty);
+                    }
+                    else
+                    {
+                        SortingNameRemovedArticles.Add(res.SelectedString);
+                        settings.GameSortingNameRemovedArticles = SortingNameRemovedArticles.ToList();

```

Comment:
This should be imo done only once when saving settings, I don't think it's necessary modifying it unless you are saving those changes. And you can use SortingNameRemovedArticles directly in FillSortingNameForAllGames.